Dockerfile

```diff
@@ -4,7 +4,6 @@ FROM golang:1.19.9 AS builder
 WORKDIR /go/src/github.com/gardener/gardener-custom-metrics
 COPY . .
 RUN make install
-# RUN CGO_ENABLED=0 GO111MODULE=on GOFLAGS=-mod=vendor go build -a -o gardener-custom-metrics.exe cmd/main.go
 
 ############# base image # TODO: Andrey: P1: Move to distroless
 FROM alpine:3.18.0 AS base
```

Comment:
Done.
Btw, I'll switch to distroless for the official release. I'm using alpine just so I'd have some troubleshooting utilities in case I need them. There's probably a better image for that purpose, but I haven't had the time to research it.

Reply:
Yes, let's switch to distroless. https://github.com/gardener/gardener-extension-registry-cache/blob/ab4928e2ebdd6cfa102b4770ac19e58cc4003e21/Dockerfile#L22
This will make the container image to run as non-root and we won't be constantly pinged by colleagues to assess CVEs in the alpine image.